go-controller/hybrid-overlay/pkg/controller/node_windows.go

```diff
@@ -52,7 +175,226 @@ func (n *NodeController) Sync(nodes []*kapi.Node) {
 
 // InitSelf initializes the node it is currently running on.
 // On Windows, this means:
-//  1. Setting up this node and its VXLAN extension for talking to other nodes
+//  1. Setting up this node and its VxLAN extension for talking to other nodes
 //  2. Setting back annotations about its VTEP and gateway MAC address to its own node object
+//  3. Initializing every VXLAN tunnels toward other
 func (n *NodeController) InitSelf() {
+	// Retrieve the host prefix from the annotations
+	hostsubnet, ok := n.thisNode.Annotations[types.HybridOverlayNodeSubnet]
+
+	if !ok {
+		klog.Errorf("Couldn't retreive the host subnet from the '%s' node's annotations", n.thisNode.Name)
+		return
+	}
+
+	baseNetwork := GetExistingNetwork(baseNetworkName, fakeSubnetCIDR, fakeSubnetGateway)
+
+	if baseNetwork == nil {
+		// Host network connectivity is temporarily lost when the first overlay network is created on Windows.
+		// This may cause disruption to other services during boot time. Once the first overlay network exists,
+		// any subsequent overlay network creation reuse the same VMSwitch and therefore don't disrupt the traffic.
+		//
+		// Overlay networks have the ability to be persistent, which means they are still present after node reboot.
+		// Endpoints and other network resources on the other hand have to be cleaned up on every reboot.
+		//
+		// In order to minimize traffic disruption but keep relying on the HNS service to clean up the unnecessary resources,
+		// we create a persistent base overlay network on top of which we create the (non-persistent) OpenShift overlay network.
+		// Endpoints are then attached to the OpenShift overlay network. The OpenShift overlay network is re-created by Hybrid-Overlay
```

Comment:
stray references to "OpenShift". (also, the comment blocks throughout the file are too wide) (as in, I should not have to scroll right in github to read the comments)